recipes/ignition-tools/all/conanfile.py

```diff
@@ -1,23 +1,25 @@
 import os
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.29.1"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir, replace_in_file, load, save
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
 
 
 class IgnitionToolsConan(ConanFile):
     name = "ignition-tools"
+    description = "Ignition entry point for using all the suite of ignition tools."
     license = "Apache-2.0"
-    homepage = "https://ignitionrobotics.org/libs/tools"
     url = "https://github.com/conan-io/conan-center-index"
-    description = "Provides general purpose classes and functions designed for robotic applications.."
-    topics = ("ignition", "robotics", "tools")
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-    generators = "cmake", "cmake_find_package_multi"
-    exports_sources = "CMakeLists.txt", "patches/**"
-    _cmake = None
+    homepage = "https://ignitionrobotics.org/libs/tools"
+    topics = ("ignition", "robotics", "tools", "gazebo")
+
+    package_type = "application"
```

Comment:
```suggestion
    package_type = "library"
```
This package contains a library. https://docs.conan.io/2/reference/conanfile/attributes.html#package-type

Reply:
It does build a library, but only as a shared library to be used dynamically in the exported `ign.rb` Ruby script. No headers are installed by the project's CMakeLists.txt.

Reply:
Yes, I read your PR description, but still, you provide a library that affects the package ID.

Reply:
Ok, thanks, I was not aware of that difference. Would the same logic not apply to any other typical application as well, though, if they package .so/.dll files to be only used by the packaged executables and not as stand-alone libraries?